src/Nethermind/Nethermind.Core/Crypto/Hash256.cs

```diff
@@ -58,6 +59,10 @@ public ValueHash256(ReadOnlySpan<byte> bytes)
             _bytes = Unsafe.As<byte, Vector256<byte>>(ref MemoryMarshal.GetReference(bytes));
         }
 
+        public ValueHash256(UInt256 uint256) : this(uint256.ToBigEndian())
```

Comment:
i think we should add `bool isBigEndian = true` parameter here

Reply:
done